test/exports/bun-exports.bun-v0.6.11.json

```diff
@@ -4452,6 +4452,8 @@
         },
         "URLSearchParams": "function",
         "Url": "function",
+        "domainToASCII": "function",
```

Comment:
Is there a reason we're updating this file? (since it's 0.6.11?)

Reply:
assumed the version was only when it was originally generated

Reply:
cc @Jarred-Sumner should this be left alone/removed in the future/kept and updated/etc ?